lib/cloudstorage/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
Do we need this ?

Reply:
Unfortunately, yes we do. It is what I am using to make calls to the GCS bucket, and it was the only one I tried that I could actually get to work without too much hassle.